apps/server/src/modules/task/uc/task.uc.spec.ts

```diff
@@ -1,27 +1,27 @@
-import { Test, TestingModule } from '@nestjs/testing';
-import { ForbiddenException, UnauthorizedException } from '@nestjs/common';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { MikroORM } from '@mikro-orm/core';
+import { ForbiddenException, UnauthorizedException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
 import { PaginationParams } from '@shared/controller';
-import { Course, Task, Lesson, User, ITaskStatus } from '@shared/domain';
-
+import { Actions, Course, ITaskStatus, Lesson, Permission, Task, User } from '@shared/domain';
+import { CourseRepo, LessonRepo, TaskRepo, UserRepo } from '@shared/repo';
 import {
-	userFactory,
 	courseFactory,
 	lessonFactory,
-	taskFactory,
-	submissionFactory,
 	roleFactory,
 	setupEntities,
+	submissionFactory,
+	taskFactory,
+	userFactory,
 } from '@shared/testing';
-import { TaskRepo, UserRepo } from '@shared/repo';
-import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { AuthorizationModule } from '@src/modules/authorization';
```

Comment:
I think it make sense that the index the module and the service exported that we can import both from this place.